src/sign/index.ts

```diff
@@ -1,6 +1,7 @@
 import { createHmac } from "crypto";
 
-export function sign(secret: string, payload: string | object): string {
+export function sign(secret?: string, payload?: string | object): string {
```

Comment:
Can you revert this change?

Reply:
> Can you revert this change?

@gr2m without this change we are not able to give coverage to this https://github.com/octokit/webhooks.js/blob/d4bd29b5718fc1390e9b0c33a7df2f340b98f721/src/sign/index.ts#L4-L6

Maybe you prefer this approach on for tests so we give coverage but we bypass the TypeScript check? https://github.com/octokit/webhooks.js/pull/237